spec/components/accordion_spec.rb

```diff
@@ -96,6 +96,33 @@ def component_name
     assert_select "h5", count: 1
   end
 
+  it "sets a default margin bottom" do
+    test_data = {
+      items: [{
+                heading: { text: "Heading 1" },
+                summary: { text: "Summary 1." },
+                content: { html: "<p>Content 1.</p>" },
+              }],
+    }
+
+    render_component(test_data)
+    assert_select '.gem-c-accordion.govuk-\!-margin-bottom-6'
```

Comment:
I would personally avoid using `govuk-!-margin-bottom-6` in a component as it adds an`!important` css.

`margin-bottom: 30px !important;`

Reply:
This is using the `shared_helper.rb`, which is used by a number of other components. The good thing about it is that if we decide to change these classes we can do it for all components that use this, by modifying that.

This seems like a wider conversation and one that shouldn't impact this PR though.

Reply:
Ok - I'll approve the PR but this is something important we should bring up with everyone because these classes could potentially cause styling issues when implemented.